src/main/java/com/timekeeper/app/dto/Employee.java

```diff
@@ -11,6 +11,19 @@
     private PayStructure payType;
     private double wage; // can be set to annual salary or hourly wage.
 
+
+    /**
+     * Sets the employee's wage after making sure the value isn't a negative number
+     * @param wage A double that reflects the employee's wage.
+     */
+    public void setWage(double wage){
+        if (wage<0.0){
+        wage = 0.0;
+        }
```

Comment:
This is a good time to throw an exception.  Tell the calling method that something is wrong.

Also, indent.  Control Alt I will do it for you automatically in IntelliJ IDEA.